examples/src/keyrings/aws_kms_keyring_example.py

```diff
@@ -0,0 +1,115 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""
+This example sets up the KMS Keyring
+"""
+import sys
+
+import boto3
+
+from aws_cryptographic_materialproviders.mpl import AwsCryptographicMaterialProviders
+from aws_cryptographic_materialproviders.mpl.config import MaterialProvidersConfig
+from aws_cryptographic_materialproviders.mpl.models import CreateAwsKmsKeyringInput
+from aws_cryptographic_materialproviders.mpl.references import IKeyring
+from typing import Dict
+
+import aws_encryption_sdk
+from aws_encryption_sdk import CommitmentPolicy
+
+
+# TODO-MPL: Remove this as part of removing PYTHONPATH hacks.
+module_root_dir = '/'.join(__file__.split("/")[:-1])
+
+sys.path.append(module_root_dir)
+
+EXAMPLE_DATA: bytes = b"Hello World"
+
+def encrypt_and_decrypt_with_keyring(
+    kms_key_id: str
+):
+    """Demonstrate an encrypt/decrypt cycle using an AWS KMS keyring."""
+
+    """
+    1. Instantiate the encryption SDK client.
+       This builds the client with the REQUIRE_ENCRYPT_REQUIRE_DECRYPT commitment policy,
+       which enforces that this client only encrypts using committing algorithm suites and enforces
+       that this client will only decrypt encrypted messages that were created with a committing
+       algorithm suite.
+       This is the default commitment policy if you were to build the client as
+       `client = aws_encryption_sdk.EncryptionSDKClient()`.
+    """
+    client = aws_encryption_sdk.EncryptionSDKClient(
+        commitment_policy=CommitmentPolicy.REQUIRE_ENCRYPT_REQUIRE_DECRYPT
+    )
+
+    """
+    2. Create boto3 clients for KMS.
+    """
+    kms_client = boto3.client('kms', region_name="us-west-2")
+
+    """
+    3. Instantiate the Material Providers
+    """
+    mat_prov: AwsCryptographicMaterialProviders = AwsCryptographicMaterialProviders(
```

Comment:
Let's move this code to just above where we create the keyring and consider it part of the "create keyring" step.
ex. https://github.com/aws/aws-encryption-sdk-java/blob/master/src/examples/java/com/amazonaws/crypto/examples/keyrings/AwsKmsRsaKeyringExample.java
Here, there isn't an MPL step, since it really isn't "interesting" to create the MPL